app/lib/providers/bangumi/search/BangumiSearchService.dart

```diff
@@ -47,15 +47,15 @@ class BangumiSearchService {
     /// does exist some results, so we need to set [chii_searchDateLine] cookie
     /// header here
     /// see https://github.com/bangumi/api/issues/43
-    String requestUrl =
+    String requestUrlStr =
```

Comment:
> https://dart.dev/guides/language/effective-dart/style#dont-use-prefix-letters

> Hungarian notation and other schemes arose in the time of BCPL, when the compiler didn’t do much to help you understand your code. Because Dart can tell you the type, scope, mutability, and other properties of your declarations, there’s no reason to encode those properties in identifier names.


Well, it's a suffix but I think this rule still applies :)

Personlly, I would suffix veriable name with type only if without the type in name variable type is confusing(i.e. `count` that is parsed from html as a string, might be better calling it `countStr`)